docs/reference/plugin/sitemap.md

```diff
@@ -0,0 +1,105 @@
+# sitemap
+
+<NpmBadge package="@vuepress/plugin-sitemap" />
+
+Generate sitemap for your vuepress project.
+
+## Install
+
+```bash
+npm i -D @vuepress/plugin-sitemap@next
+```
+
+## Frontmatter Options
+
+### sitemap.changefreq
+
+- Type: `'always'|'hourly'|'daily'|'weekly'|'monthly'|'yearly'|'never'`
+
+- Default: `'daily'`
+
+- Details:
+
+  Page default update frequency.
```

Comment:
More description about the differences between frontmatter and options, i.e. should specify that frontmatter would override options in current page.

Also, should add reference to corresponding options, and vice versa.